docs/source/keywords.rst

```diff
@@ -599,6 +604,11 @@ Kokkos will attempt to autodetect the architecture flag at configuration time.
       * GFX906
       * MI50, MI60
       * (since Kokkos 4.2)
+
+    * * ``Kokkos_ARCH_AMD_GFX1103``
+      * GFX1103
+      * Phoenix APU
```

Comment:
As I understand it, "Phoenix" is the codename for the microarchitecture whereas this column says "Models".
Should we list a couple popular models, something like "Phoenix APUs (e.g. MODEL-1, MODEL-2, etc.)"

Reply:
I have added something